platforms/platforms.go

```diff
@@ -257,10 +257,5 @@ func Format(platform specs.Platform) string {
 func Normalize(platform specs.Platform) specs.Platform {
 	platform.OS = normalizeOS(platform.OS)
 	platform.Architecture, platform.Variant = normalizeArch(platform.Architecture, platform.Variant)
-
-	// these fields are deprecated, remove them
```

Comment:
I guess the point of not using these fields was that in order to actually make platforms containing these fields work containerd should define how they are serialized when `Format()` and `Parse()` of this package are called.

Reply:
We were discussing Parse this a bit here: https://github.com/containerd/containerd/pull/6491#discussion_r796835311